modules/apps/object/object-test/src/testFunctional/macros/json/JSONObject.macro

```diff
@@ -108,37 +108,68 @@ definition {
 			var userPassword = "test";
 		}
 
+		var entryLanguage = ${entryLanguage};
 		var objectName = StringUtil.toLowerCase(${objectName});
 		var list1 = ListUtil.newListFromString(${fieldNameList});
 		var list2 = ListUtil.newListFromString(${valueList});
+		var list3 = ListUtil.newListFromString(${entryLanguageList});
 		var size = ListUtil.size(${list1});
+		var size2 = ListUtil.size(${list3});
 		var fieldName = ListUtil.get(${list1}, 0);
 		var value = ListUtil.get(${list2}, 0);
-
+		var language = ListUtil.get(${list3}, 0);
 		var data = "\"${fieldName}\": \"${value}\"";
+		var data2 = "\"${language}\": \"${value}\"";
```

Comment:
if you're going to name it `data2`, then `data` must be `data1`. That's something BChan enforces.

`Where there is 2, there must be 1`.